lib/core/network/models/user_profile_data.dart

```diff
@@ -20,4 +20,7 @@ class UserProfileData with _$UserProfileData {
   factory UserProfileData.fromJson(Map<String, dynamic> json) => _$UserProfileDataFromJson(json);
 
   String get userNameOrAccount => userName != null ? userName! : accountName;
+
+  bool get isEmpty =>
```

Comment:
can we name this something more meaningful? 
Like hasUserProfileFromNightmareAwsService?

Reply:
Hmm but I don't want to invert the boolean... 

hasUserProfile would be ==  !isEmpty

Reply:
renamed to doesNotHaveUserProfileOnPPPService